src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -244,6 +245,173 @@ function RemoteFunctions(experimental) {
                 return;
             }
 
+            var setVisibility = function (el){
+                if (
+                    !config.remoteHighlight.showPaddingMargin || 
+                    parseInt(el.height, 10) <= 0 || 
+                    parseInt(el.width, 10) <= 0 
+                ) {
+                    el.display = 'none';
+                } else {
+                    el.display = 'block';
+                }
+            };
+            
+            var elementStyling = window.getComputedStyle(element, null);
+            
+            var calculateSize = function () {
+                var sum = 0;
+                var i;
+                for (i = 0; i < arguments.length; i++) {
```

Comment:
`calculateSize` should probably return value in pixels too instead of using string concat on every use.